src/components/pages/TeamPage.jsx

```diff
@@ -0,0 +1,126 @@
+
+import React from 'react';
+import GreenBlobImage from '../../assets/blob_green.png';
+import PurpleBlobImage from '../../assets/blob_dusty_purple2.png';
+import YellowBlobImage from '../../assets/blob_yellow.png';
+import teamMembers from '../../data/teamData';
+import ScrollToTop from '../layouts/ScrollToTop';
+
+
+const TeamPage = () => {
+
+  return (
+    <>
+      <section className="flex flex-col items-center justify-center">
```

Comment:
Lots of repetition here. Let's map over the data and render a component that renders the styles and data for each team member card.

Reply:
MentorCard and MemberCard components were created and the TeamPage component was refactored